test/evaluation/subjectIsIssuerEvaluationHandler.spec.ts

```diff
@@ -46,8 +49,15 @@ describe('evaluate', () => {
         'id': 'https://www.w3.org/TR/vc-data-model/#types'
       }
     ];
-    subjectIsIssuerEvaluationHandler.verifiablePresentation = new VP(new Presentation([], presentationSubmission, [], verifiableCredential, null, null));
-    subjectIsIssuerEvaluationHandler.handle(pdSchema, new VP(vpSimple));
+    subjectIsIssuerEvaluationHandler.verifiablePresentation =  {
+      '@context': [],
+      type: null,
+      presentationSubmission,
+      verifiableCredential,
+      holder: null,
+      proof: null
+    };
```

Comment:
instantiation should be cleaner with new

Reply:
Nope it is not. You are looking at this with a Java hat on